ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/433__data-science-pipelines-general.robot

```diff
@@ -0,0 +1,80 @@
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataSciencePipelines.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource            ../../../Resources/Page/Operators/OpenShiftPipelines.resource
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+Suite Setup         General Suite Setup
+Suite Teardown      General Suite Teardown
+
+
+*** Variables ***
+${PROJECT_USER2}=    dspa-project-user2
+${PROJECT_USER3}=    dspa-project-user3
+${S3_BUCKET}=    ods-ci-ds-pipelines
+
+
+*** Test Cases ***
+Verify Ods User Can Bind The Route Role
+    [Documentation]    Verify Ods User Can Bind The Route Role
+    [Tags]      Sanity
+    ...         Tier1
+    ...         ODS-2209
+    Create A Data Science Projects    ${TEST_USER_2.USERNAME}    ${TEST_USER_2.PASSWORD}    ${TEST_USER_2.AUTH_TYPE}    ${PROJECT_USER2}    # robocop: disable:line-too-long
+    Create A Data Science Projects    ${TEST_USER_3.USERNAME}    ${TEST_USER_3.PASSWORD}    ${TEST_USER_3.AUTH_TYPE}    ${PROJECT_USER3}    # robocop: disable:line-too-long
+    ${status}    Login And Wait Dsp Route    ${TEST_USER_2.USERNAME}    ${TEST_USER_2.PASSWORD}    ${PROJECT_USER3}    ds-pipeline-pipelines-definition    ${1}    # robocop: disable:line-too-long
+    Should Be True    ${status} == 403    The user must not have permission to access
+    Run    oc policy add-role-to-user ds-pipeline-user-access-pipelines-definition ${TEST_USER_2.USERNAME} -n ${PROJECT_USER3} --role-namespace=${PROJECT_USER3}    # robocop: disable:line-too-long
+    ${status}    Login And Wait Dsp Route    ${TEST_USER_2.USERNAME}    ${TEST_USER_2.PASSWORD}    ${PROJECT_USER3}    ds-pipeline-pipelines-definition    ${1}    # robocop: disable:line-too-long
+    Should Be True    ${status} == 200    Rolling Binding Not Working    # robocop: disable:line-too-long
+
+
+*** Keywords ***
+General Suite Setup
+    [Documentation]    Suite setup steps for testing DSG. It creates some test variables
+    ...                and runs RHOSi setup
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+
+General Suite Teardown
+    [Documentation]    General Suite Teardown
+    Remove Pipeline Project    ${PROJECT_USER2}
+    Remove Pipeline Project    ${PROJECT_USER3}
+    RHOSi Teardown
+
+Create A Data Science Projects
+    [Documentation]    Create A Data Science Projects
+    [Arguments]     ${user}    ${password}    ${auth_type}    ${project}
+    Launch Data Science Project Main Page    username=${user}
+    ...    password=${password}
+    ...    ocp_user_auth_type=${auth_type}
+    ...    browser_alias=${user}-session
+    Open Data Science Projects Home Page
```

Comment:
it shouldn't be needed, `Launch Data Science Project Main Page` already opens the home page

Reply:
Ok